plugins/postgres/dbt/adapters/postgres/__init__.py

```diff
@@ -2,6 +2,7 @@
 from dbt.adapters.postgres.connections import PostgresConnectionManager  # noqa
 from dbt.adapters.postgres.connections import PostgresCredentials
 from dbt.adapters.postgres.relation import PostgresColumn  # noqa
+from dbt.adapters.postgres.relation import PostgresRelation
```

Comment:
```suggestion
from dbt.adapters.postgres.relation import PostgresRelation  # noqa: F401
```

The import is, in fact used in `postgres/impl.py`. flake8 just isn't aware.

Reply:
noqa added